source/build/Linux/TestAppLinux.cpp

```diff
@@ -13,10 +13,13 @@ namespace PlayFabUnit
         // Get the path to the test data file from the current environment vars.
         std::string titleDataPath;
         {
-            char* titleDataPathCStr =  getenv("PF_TEST_TITLE_DATA_JSON");
+            char* titleDataPathCStr = getenv("PF_TEST_TITLE_DATA_JSON");
 
             if (titleDataPathCStr == nullptr)
+            {
+                printf("Could not load PF_TEST_TITLE_DATA_JSON var: %s\n", titleDataPathCStr);
```

Comment:
Should we consider the secure version of printf? Or if this is a shipped test it doesn't really matter, this will handle any linux terminal printing.

Reply:
TestApp. Not part of the official SDK.
If we can do it in 30 seconds, sure. Otherwise, I don't care enough.